superset/db_engine_specs/ocient.py

```diff
@@ -14,10 +14,13 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-from typing import Optional, List
+from typing import Any, Dict, List, Optional, Pattern, Tuple, TYPE_CHECKING
+import re
 
 from sqlalchemy.engine.reflection import Inspector
 from superset.db_engine_specs.base import BaseEngineSpec
+from superset.errors import SupersetErrorType
+from flask_babel import gettext as __
```

Comment:
unused import?

Reply:
Used in OcientEngineSpec > custom_errors. Really stupid import name, but all other dialects use the same convention